### PR TITLE
Add thunderhub

### PR DESCRIPTION
**Pull request by krtk6160**

### CODE REVIEW

File: charts/lnd/Chart.yaml [hunk 1]
```diff
@@ -16,8 +16,10 @@ engine: gotpl
 dependencies:
   - name: loop
     version: x.x.x
-  - name: rtl
+    condition: loop.enabled
```
Comment (nicolasburtey) [inline charts/lnd/Chart.yaml:19]: didn't know we could do that. is that a new feature of helm?

File: charts/lnd/charts/thunderhub/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: ACCOUNT_CONFIG_PATH
+            value: "/home/thunderhub.yaml"
+          - name: LOG_LEVEL
+            value: 'debug'
+          - name: YML_ENV_1
```
Comment (nicolasburtey) [inline charts/lnd/charts/thunderhub/templates/deployment.yaml:38]: we could use a better name

**Review by nicolasburtey — COMMENTED** [review]
could we keep rtl and have both as an option?

rtl can be helpful if we ever want to have c-lightning as an additional lightning node option. also I've never tried thunderhub on my side so having both can be helpful for now to see how to best use thunderhub.

I know there was some auth/secret hurdle with rtl maybe this is why you have deleted it?

File: charts/lnd/Chart.yaml (hunk 1 above)
Comment (krtk6160) [reply charts/lnd/Chart.yaml:19]: Afaik it was there in helm v2 also, but the way to achieve it was a bit different. This was what I was referring to when we had the discussion over subchart enabling/disabling.

File: charts/lnd/charts/thunderhub/templates/deployment.yaml (hunk 2 above)
Comment (krtk6160) [reply charts/lnd/charts/thunderhub/templates/deployment.yaml:38]: Thunderhub requires us to use this env var name.

**Comment by krtk6160** [comment]
> could we keep rtl and have both as an option?

> rtl can be helpful if we ever want to have c-lightning as an additional lightning node option. also I've never tried thunderhub on my side so having both can be helpful for now to see how to best use thunderhub.
> 

Okay.

> I know there was some auth/secret hurdle with rtl maybe this is why you have deleted it?

No, I deleted it because I thought we wanted to switch to thunderhub.

File: charts/lnd/charts/thunderhub/templates/deployment.yaml (hunk 2 above)
Comment (nicolasburtey) [reply charts/lnd/charts/thunderhub/templates/deployment.yaml:38]: ok

**Review by nicolasburtey — APPROVED** [review]

**Pull request merged**